.github/workflows/run_tests.yml

```
@@ -55,15 +55,14 @@ jobs:
         pip install -r requirements-tests.txt
         
     - name: Run pytest with codecoverage
-      run: |
-        coverage run --source=vermouth $(which pytest) -vv vermouth --hypothesis-show-statistics
-        coverage report --omit='*/bin/pytest'
-        
+      run:  pytest --cov vermouth --hypothesis-show-statistics
```

```suggestion
      run:  pytest vermouth --cov=vermouth --hypothesis-show-statistics
```

This incantation works on my machine at least...